tests/forms_tests/tests/test_forms.py

```diff
@@ -5329,3 +5386,31 @@ class CustomFrameworkForm(FrameworkForm):
             '<label for="id_name" class="required">Name:</label>'
             '<legend class="required">Language:</legend>',
         )
+
+    def test_custom_boundfield(self):
+        class CustomBoundField(BoundField):
```

Comment:
We should also add a test for the case then it's both set on the form itself and when instantiating the form. Which is given priority in this case?

``` python
class TestForm(Form):
        ...
        bound_field_class = BoundFieldWithoutColon

form = TestForm(bound_field_class=AnotherBoundFieldClass)
```